cmake/SetupCompilerOptions.cmake

```diff
@@ -90,6 +90,10 @@ elseif("${CMAKE_CXX_COMPILER_ID}" MATCHES "HCC") # For ROCm
     set(COMPILER_FAMILY_IS_HCC 1)
     message(STATUS "Compiler family is HCC Clang")
     
+elseif("${CMAKE_CXX_COMPILER_ID}" MATCHES "PGI")
```

Comment:
Does CMake correctly identify PGI?

Reply:
I'm using cmake 3.8.1 and it seems to:

...
-- The CXX compiler identification is PGI 18.1.1
-- The C compiler identification is PGI 18.1.1
-- The Fortran compiler identification is PGI 18.1.1
...

Reply:
Perfect!

Reply:
For reference: https://cmake.org/cmake/help/v3.8/variable/CMAKE_LANG_COMPILER_ID.html